content/documentation/rpc/chain/broadcast_transaction.mdx

```diff
@@ -25,4 +25,4 @@ Returns accepted: true if the transaction was accepted into the node's mempool o
 }
 ```
 
-### [View on Github](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/routes/chain/broadcastTransaction.ts)
+###### [View on Github](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/routes/chain/broadcastTransaction.ts)
```

Comment:
This is fine to have these in here, but it might have been easier to review as a separate change 😁

Reply:
will definitely keep that in mind for the next time